src/network.cpp

```diff
@@ -10,7 +10,7 @@ namespace net {
 std::pair<std::uint32_t, std::uint32_t> ip_block(const std::string_view &ip);
 
 std::vector<std::pair<std::uint32_t, std::uint32_t>> pc_ips {
-  ip_block("127.0.0.1/32"sv)
+  ip_block("127.0.0.0/8"sv)
```

Comment:
This was a little drive-by fix. IPv4 loopback is the whole `127.0.0.0/8` subnet, not just `127.0.0.1`.

Reply:
I love that we have ~16 million loopback addresses. lol